typescript/infra/config/contexts.ts

```diff
@@ -1,6 +1,6 @@
 // All valid deployment contexts. Environments may use just a subset of these contexts.
 export enum Contexts {
-  Abacus = 'abacus',
+  Hyperlane = 'abacus', // TODO rename when infra changes
```

Comment:
I'm a little worried this can get confusing, I wonder if we should leave the key name as `Abacus` given we're leaving the value as `'abacus'`?

The context name is used for naming / looking up the names of things in k8s or secrets in GCP secret manager. Feel free to push back tho

Reply:
Agreed, will fix